### PR TITLE
Radiot Suomi Presence

### PR DESCRIPTION
**Pull request by TehRamsus**
kuva
kuva

### CODE REVIEW

File: websites/R/Radiot-Suomi/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        "id": "199455497533784064"
+    },
+    "service": "Radiot Suomi",
+    "description": {
+        "fi": "Voit kuunnella suomalaisia radiokanavia internetissä!",
+        "en": "You can listen to Finnish radio channels on the Internet!"
+    },
+    "url": [
+        "www.radio-suomi.com"
+    ],
```
Comment (Bas950) [inline websites/R/Radiot-Suomi/dist/metadata.json:13]: Doesnt need to be in an array if its only 1

File: websites/R/Radiot-Suomi/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    "version": "1.2.3",
+    "logo": "http://www.radio-suomi.com/static/icons/App_512x512/fi.png",
```
Comment (Bas950) [inline websites/R/Radiot-Suomi/dist/metadata.json:15]: version 1.0.0 on first release
and upload logo to imgur

File: websites/R/Radiot-Suomi/presence.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
...QUFDLEVBQUU7WUFDN0QsWUFBWSxDQUFDLGNBQWMsR0FBRyxhQUFhLENBQUM7WUFDNUMsWUFBWSxDQUFDLE9BQU8sR0FBRyxRQUFRLENBQUMsYUFBYSxDQUFDLFlBQVksQ0FBQyxDQUFDLFdBQVcsQ0FBQztZQUN4RSxZQUFZLENBQUMsS0FBSyxHQUFHLFlBQVksUUFBUSxDQUFDLGFBQWEsQ0FBQyxjQUFjLENBQUMsQ0FBQyxXQUFXLEVBQUUsQ0FBQztZQUN0RixZQUFZLENBQUMsYUFBYSxHQUFHLFlBQVksQ0FBQztTQUUzQzthQUFNLElBQUksUUFBUSxDQUFDLFFBQVEsQ0FBQyxRQUFRLENBQUMsUUFBUSxDQUFDLGNBQWMsQ0FBQyxFQUFFO1lBQzlELFlBQVksQ0FBQyxjQUFjLEdBQUcsYUFBYSxDQUFDO1lBQzVDLFlBQVksQ0FBQyxPQUFPLEdBQUcsUUFBUSxDQUFDLGFBQWEsQ0FBQyxZQUFZLENBQUMsQ0FBQyxXQUFXLENBQUM7WUFDeEUsWUFBWSxDQUFDLEtBQUssR0FBRyxZQUFZLFFBQVEsQ0FBQyxhQUFhLENBQUMsY0FBYyxDQUFDLENBQUMsV0FBVyxFQUFFLENBQUM7WUFDdEYsWUFBWSxDQUFDLGFBQWEsR0FBRyxXQUFXLENBQUM7U0FFMUM7YUFBTSxJQUFJLFFBQVEsQ0FBQyxRQUFRLENBQUMsUUFBUSxDQUFDLFFBQVEsQ0FBQyxlQUFlLENBQUMsRUFBRTtZQUMvRCxZQUFZLENBQUMsY0FBYyxHQUFHLGFBQWEsQ0FBQztZQUM1QyxZQUFZLENBQUMsS0FBSyxHQUFHLFdBQVcsQ0FBQztZQUNqQyxZQUFZLENBQUMsYUFBYSxHQUFHLGVBQWUsQ0FBQztTQUU5QztLQUNGO0lBRUQsSUFBSSxZQUFZLENBQUMsT0FBTyxJQUFJLElBQUksRUFBRTtRQUNoQyxRQUFRLENBQUMsWUFBWSxFQUFFLENBQUM7UUFDeEIsUUFBUSxDQUFDLFdBQVcsRUFBRSxDQUFDO0tBQ3hCO1NBQU07UUFDTCxRQUFRLENBQUMsV0FBVyxDQUFDLFlBQVksQ0FBQyxDQUFDO0tBQ3BDO0FBQ0gsQ0FBQyxDQUFDLENBQUMifQ==
```
Comment (Bas950) [inline websites/R/Radiot-Suomi/presence.ts:31]: how did you manage to get a sourcemapping url in your ts file....

**Review by Bas950 — CHANGES_REQUESTED** [review]
Fix codacy issues

**Comment by TehRamsus** [comment]
Fixed SourceMappingURL, fixed array, changed version to 1.0.0 and added logo to imgur.

**Comment by Bas950** [comment]
well you did something wrong.... you made a new metadata file outside of the dist folder...

**Comment by TehRamsus** [comment]
oh fuck accidentally, sorry :D i'll fix that xD

File: websites/R/Radiot-Suomi/dist/metadata.json [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        "id": "199455497533784064"
+    },
+    "service": "Radiot Suomi",
+    "description": {
+        "fi": "Voit kuunnella suomalaisia radiokanavia internetissä!",
+        "en": "You can listen to Finnish radio channels on the Internet!"
+    },
+    "url": "www.radio-suomi.com",
+    "version": "1.2.3",
+    "logo": "https://i.imgur.com/nhxjVmG.png",
```
Comment (Bas950) [inline websites/R/Radiot-Suomi/dist/metadata.json:13]: version 1.0.0 on release

File: websites/R/Radiot-Suomi/presence.ts [hunk 5]
```diff
@@ -0,0 +1,30 @@
+let presence = new Presence({
+    clientId: "727550263786995792"
+});
+let browsingStamp = Math.floor(Date.now() / 1000);
```
Comment (Bas950) [inline websites/R/Radiot-Suomi/presence.ts:4]: const and merge the consts

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Alanexei — CHANGES_REQUESTED** [review]
Please resolve the Codacy issues.

**Review by Bas950 — APPROVED** [review]

**Review by Alanexei — APPROVED** [review]

**Pull request merged**